src/Arcus.WebApi.OpenApi.Extensions/OAuthAuthorizeOperationFilter.cs

```diff
@@ -33,9 +36,13 @@ public OAuthAuthorizeOperationFilter(IEnumerable<string> scopes)
         /// <summary>
         /// Applies the OperationFilter to the API <paramref name="operation"/>.
         /// </summary>
```

Comment:
Do we use this approach, or just provide 2 overloads which are wrapped in the directives?

Reply:
We use either one when the project is build for either one of the target frameworks.